pymove/_version.py

```diff
@@ -350,8 +313,7 @@ def plus_or_dot(pieces):
 
 
 def render_pep440(pieces):
-    """
-    Build up version string, with post-release "local version identifier".
+    """Build up version string, with post-release "local version identifier".
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Multi-line docstring summary should start at the second line (D213)](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=5497268)